tailwind.config.js

```diff
@@ -41,27 +41,6 @@ export default {
           quote: '#58cef4',
         },
       },
-      // Content in the sidebar should never exceed a max-width of `768px`, and
-      // that content should be auto-centered
-      // See https://tailwindcss.com/docs/container
-      container: {
-        center: true,
-        // Horizontal padding is larger for wider screens
-        padding: {
-          // Precise horizontal padding for annotation-card alignment
-          DEFAULT: '9px',
-          lg: '4rem',
-        },
-        // By default, tailwind will provide appropriately-sized containers at
-        // every breakpoint available in `screens`, but for the sidebar, only
-        // one width matters: the width associated with the `lg` breakpoint.
-        // The content container should never be larger than that. `container`
-        // has a `max-width:100%` until the `lg` breakpoint, after which it
-        // never exceeds `768px`.
-        screens: {
-          lg: '768px',
-        },
-      },
```

Comment:
Move to a utility defined in CSS. I have simplified the definition there as well.